src/app/pages/viewing/responses-list/responses-list.component.css

```diff
@@ -188,48 +191,108 @@
   }
 }
 
+/*noinspection CssUnusedSymbol*/
```

Comment:
Comment why it's being suppressed. Same goes for other instances of suppressions here.